basicswap/static/js/modules/coin-manager.js

```diff
@@ -0,0 +1,230 @@
+const CoinManager = (function() {
+    const coinRegistry = [
+        {
+            symbol: 'BTC',
+            name: 'bitcoin',
+            displayName: 'Bitcoin',
+            aliases: ['btc'],
```

Comment:
I'm not sure the use here, but for consistency


```suggestion
            aliases: ['btc', 'bitcoin'],
```